src/composables/fungibleTokens.ts

```diff
@@ -0,0 +1,511 @@
+import camelcaseKeysDeep from 'camelcase-keys-deep';
+import { SCHEMA } from '@aeternity/aepp-sdk';
+import FUNGIBLE_TOKEN_CONTRACT from 'aeternity-fungible-token/FungibleTokenFullInterface.aes';
+import BigNumber from 'bignumber.js';
+import pairInterface from 'dex-contracts-v2/build/IAedexV2Pair.aes';
+import {
+  computed,
+  ref,
+  watch,
+} from '@vue/composition-api';
+import { isEmpty, uniqBy } from 'lodash-es';
+import {
+  AETERNITY_CONTRACT_ID,
+  AETERNITY_DECIMALS,
+  AETERNITY_NAME,
+  AETERNITY_SYMBOL,
+  aettosToAe,
+  calculateSupplyAmount,
+  categorizeContractCallTxObject,
+  convertToken,
+  CURRENCY_URL,
+  fetchJson,
+  handleUnknownError,
+  watchUntilTruthy,
+  ZEIT_TOKEN_INTERFACE,
+} from '../popup/utils';
+import type {
+  IDefaultComposableOptions,
+  IDexContracts,
+  IDexTokenPairs,
+  INetwork,
+  IPendingTransaction,
+  IRawTokenTransaction,
+  IToken,
+  ITokenList,
+  ITransaction,
+  IAsset,
+  Balance,
+  IAex9Balance,
+  ITokenBalance,
+} from '../types';
+import { createPollingBasedOnMountedComponents } from './composablesHelpers';
+import { useSdk } from './sdk';
+
+const activeAddress = ref<string | null>(null);
+const availableTokens = ref<ITokenList>({});
```

Comment:
I think the available tokens should be cached for each of the addresses. So here it should be:
```
const availableTokens = ref<Record<string, ITokenList>({});
```
where the `string` is in account address.

Reply:
`export type ITokenList = Record<string, IToken>`